### DISCUSSION TITLE
Packets Belated

### DISCUSSION THREAD

**Discussion by video2go** [Haivision/srt#2155] [Q&A] [answered]
Hi,
what does packets Belated mean? What I have not found what causes packets Belated and NAKs received. When streaming I can see that packets bleated grows slowly, but the stream itself is OK. I would like to tune my settings, but maxbw and latency does not make a difference. Of what of is this "packets Belated, NAKs received" depended? 

I use SRT to stream over bonded network with openMPTCProuter to my VPS. This stream is without any issue, no packets Bleated or similar. 

1. HW Encoder to Raspberry Pi via local LAN (Pi is running Nimble). 
2. From Nimble on Pi to VPS Nimble service (over bonded network)
3. From VPS to RTMP (restream) and to office installation of private server running Nimble. VPS connection is 2G, My office is 200/200. Here I get a lot of packets Belated. I would like to fine tune this. Possible? How?
best
grega

**Comment by maxsharabayko** [comment, marked as answer]
Hi @video2go 
Belated packets are those received but IGNORED due to having arrived too late. The payload of those packets may have been dropped already by the receiver in favor of the following packets, or actually already recovered by a previous retransmission attempt.
For more info please refer to the [SRT Statistics](https://github.com/Haivision/srt/blob/v1.4.4/docs/API/statistics.md#pktRcvBelated).
